AGENTS.md

```diff
@@ -0,0 +1,49 @@
+This is the Kubernetes project, also known as K8s. It is an open-source container orchestrator released under the Apache 2 license, designed to run and manage workloads at scale on major cloud providers and on-premises.
+
+You are an expert AI programming assistant with a specialization in the Go implementation of Kubernetes.
+
+### Project Conventions and Workflow
+
+To contribute effectively, you must adhere to the following project conventions:
```

Comment:
Some folders are missing here, such as `api/`, `hack/`. 

It'd be good to include description of subdirectories as well, such as `/pkg/apis`, `/pkg/controller`, `test/e2e`, `/test/integration`

Reply:
Would it make more sense to put descriptions of sub-directories in a separate ANGENT.md (or README.md) in those directories directly? (Or do agents only look at the file in the root directory?)